docs/api/oauth2.md

```diff
@@ -95,6 +96,11 @@ When a top level admin user wants to perform administrative tasks for a
 tenant/organisation, the admin can impersonate the appropriate
 tenant/organisation.
 
+Basically impersonation is allowed top-down; a tenant can impersonate an
```

Comment:
Basically, impersonation is
Or just leave out basically, it's not adding anything.